### DISCUSSION TITLE
Parameter Sweep of Varactor

### DISCUSSION THREAD

**Discussion by tomhajjar** [ra3xdh/qucs_s#771] [Q&A] [answered]
I'm not having luck getting a varactor tuned BPF to work in ngspice. Works under Qucsator. I revisited the project in #768 looking for ideas but I couldn't get that project to work either.

I don't need to use a Nutmeg script to get this to work do I?

![2024-06-19_000142](https://github.com/ra3xdh/qucs_s/assets/102838604/3c0c1e4a-5443-4293-981d-4ed69c6a4eef)
![2024-06-19_000206](https://github.com/ra3xdh/qucs_s/assets/102838604/e3754893-6c10-4670-9932-90a257f53472)

[VCO_prj.zip](https://github.com/user-attachments/files/15895275/VCO_prj.zip)

**Comment by dwarning** [comment]
Something went wrong.
If I change in diagram properties v(vout) to ac.v(vout) the curves are as they should.

**Comment by ra3xdh** [comment, marked as answer]
It is not a bug. You forgot to change the output variables after changing the simulator. Probably you forgot to switch the simulator to Ngspice too. After switching the simulator to Ngspice and selection of the correct output variable name everything works as expected. Also keep in mind that Qucsator doesn't accept the source name as the sweep parameter name. The schematic requires the following modification to works with Ngspice/Qucsator seamless. 

![image](https://github.com/ra3xdh/qucs_s/assets/4920080/54191236-750c-4636-a9a6-4111adba8223)
